Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -345,6 +349,7 @@ void Renderer::DrawDebugText()
                              std::lround(SConfig::GetInstance().m_EmulationSpeed * 100.f)),
         std::string("Copy XFB: ") + xfbcopy_text +
             (g_ActiveConfig.bImmediateXFB ? " (Immediate)" : ""),
+        std::string("Volume: ") + audio_text,
```

Comment:
Do you *need* the std::string here? If not, could you remove it?

Reply:
![block of osd messages](https://user-images.githubusercontent.com/38867833/41032520-eed27ace-6983-11e8-92ba-1bc5b06f8903.png)

Well, I made the Volume message part of this block of OSD messages because those were the only settings related hotkeys that gave OSD messages.

If I just remove that line then I don't get my Volume message. Of course I could make the message not part of that block but then I'd need to make a copy of Renderer::DrawDebugText() with just the Volume message. 

If you really don't want it there then I can do that.

Reply:
leoetlino isn't suggesting to remove the whole line, just changing `std::string("Volume: ")` to `"Volume: "`.

Reply:
Oh...

Ooops